mlflow/ml-package-versions.yml

```
@@ -548,6 +551,9 @@ transformers:
   autologging:
     minimum: "4.25.1"
     maximum: "4.37.1"
+    unsupported: [
+        "4.38.1", # to avoid this patch: https://github.com/huggingface/transformers/pull/29032
```

```suggestion
    unsupported: [
        # to avoid this patch: https://github.com/huggingface/transformers/pull/29032
        "4.38.0",
        "4.38.1",
```

It seems 4.38.0 has the same issue.